### PR TITLE
DIO-757: [fix case where no endpoint exists yet]

### PR DESCRIPTION
**Pull request by opalampo**
Needs to be merged along with DIO-786 in the wordpress-theme-dorel-juvenile

### CODE REVIEW

File: fields/acf-rest-v5.php [hunk 1]
```diff
@@ -14,205 +14,205 @@
...
+        /*
+        *  __construct
+        *
+        *  This function will setup the field type data
+        *
+        *  @type	function
+        *  @date	5/03/2014
+        *  @since	5.0.0
+        *
+        *  @param	n/a
```
Comment (mrpharderwijk) [inline fields/acf-rest-v5.php]: $settings is a param for the construct right?

File: fields/acf-rest-v5.php [hunk 2]
```diff
@@ -14,205 +14,205 @@
...
+        *
+        *  Create the HTML interface for your field
+        *
+        *  @param	$field (array) the $field being rendered
+        *
+        *  @type	action
+        *  @since	3.6
+        *  @date	23/01/13
+        *
+        *  @param	$field (array) the $field being edited
```
Comment (mrpharderwijk) [inline fields/acf-rest-v5.php]: two @param documented

File: fields/acf-rest-v5.php (hunk 1 above)
Comment (opalampo) [inline fields/acf-rest-v5.php]: It is injected as a parameter to the constructor and assigned to the instance variable. Do you see something wrong that I am missing or was it just a clarification?

File: fields/acf-rest-v5.php (hunk 2 above)
Comment (opalampo) [inline fields/acf-rest-v5.php]: removed the redundant one

**Pull request merged**